app/views/content/_actions.html.erb

```diff
@@ -0,0 +1,46 @@
+<!-- draft mode -->
+<% if action_name.in? %w{edit layout annotate} %>
+  <% if @content.is_a? Content::Casebook %>
```

Comment:
This too, could be a series of methods on a presenter object:

```ruby
class SomePresenter
  attr_reader :content
  def initalize(content)
    @content = content
  end
  
  def action_content
    if casebook?
      casebook_actions
    elsif section?
      section_actions
      # etc...
    end
  end

  def casebook?
    @content.is_a(Content::Casebook)
  end

  # etc...

  def casebook_actions
    # these html things will require some library to access them, otherwise it's a huge pain
    button_to t('content.actions.publish'), casebook_path(@casebook), method: :patch, params: {content_casebook: {public: true}}, class: 'action publish one-line'
    link_to t('content.actions.preview'), casebook_path(@casebook), class: 'action one-line preview'
    link_to t('content.actions.add-resource'), new_section_path(@casebook), class: 'action add-resource'
    button_to t('content.actions.add-section'), sections_path(@casebook, params: {parent: @section.try(:id)}), method: :post, class: 'action add-section'
  end

  # etc...
end
```